source/mesh_deformation/fastscape.cc

```diff
@@ -1327,6 +1327,16 @@ namespace aspect
               unsigned int op_side = index_top;
               int jj = fastscape_nx;
 
+              // Set top ghost node
```

Comment:
Just to understand: The main difference of this change is that if we hit the `else continue;` statement below, this operation is done in any case, while previously it would only be done if we do not trigger that `else`, is this correct? Can you extend the comment in in line 1330 and in line 1335 to explain what you are doing in these lines, and why it is necessary?

E.g. I am not exactly sure what you are setting here. The comment says `Set top ghost node` but what are you setting it to? I suspect the velocity of the inner node which is actually the top boundary node of the ASPECT mesh, is this correct? And the number `2*fastscape_nx` is also slightly mysterious, can you explain in the comment why we add or subtract that number?